share/p2p/peers/options.go

```diff
@@ -56,6 +59,7 @@ func DefaultParameters() Parameters {
 		// sync time for large blocks. This value gives our (discovery) peers enough time to sync
 		// the new block before we ask them again.
 		PeerCooldown: 3 * time.Second,
+		StoreWindow:  50,
```

Comment:
Also, why do you think this needs to be configured by the node operator?

Reply:
I don't think node operator needs to, but I think it is nice to have it exposed for testing

Reply:
Also might worth removing it just so it doesn't break config. Sucks that adding new fields to any subcomponent configs or changes of any default values requires re-init of the node.

Reply:
removed from config, don't wanna break